instrumentation/base/lib/opentelemetry/instrumentation/version.rb

```diff
@@ -6,6 +6,6 @@
 
 module OpenTelemetry
   module Instrumentation
-    VERSION = '0.20.0'
+    VERSION = '0.21.0'
```

Comment:
The instrumentations all have pessimistic version dependencies on Base and will only expect a bump on bug fix versions. Either we stick with that or change all of those constraints and bump them in the instrumentation gemspecs. 

```suggestion
    VERSION = '0.20.1'
```